bin/api-docs/are-readmes-unstaged.js

```diff
@@ -0,0 +1,39 @@
+#!/usr/bin/env node
+
+/**
+ * Node dependencies.
+ */
+const { join } = require( 'path' );
+const chalk = require( 'chalk' );
+const execSync = require( 'child_process' ).execSync;
+
+/**
+ * Local dependencies.
+ */
+const getPackages = require( './packages' );
+
+const getUnstagedFiles = () => execSync( 'git diff --name-only', { encoding: 'utf8' } ).split( '\n' ).filter( ( element ) => '' !== element );
+
+const readmeFiles = getPackages().map( ( [ packageName ] ) => join( 'packages', packageName, 'README.md' ) );
+const unstagedFiles = getUnstagedFiles();
+
+const unstagedReadmes = [];
+unstagedFiles.forEach( ( element ) => {
+	if ( readmeFiles.includes( element ) ) {
+		unstagedReadmes.push( element );
+	}
+} );
```

Comment:
This kinda looks like a `Array#filter` ?

```js
const unstagedReadmes = unstagedFiles.filter( ( element ) => readmeFiles.includes( element ) );
```

Reply:
The core is now more compact, thanks! Added in https://github.com/WordPress/gutenberg/pull/18820/commits/f4aa5ff9bf6ddaae7929e7a1169184dfb31b0382